modules/react-components/stories/assets/images/icons/controls-icon.svg

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+
+<!--
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+-->
+
+<svg version="1.1" class="icon" id="controls-icon" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 width="14px" height="14px" viewBox="0 0 54 54" xml:space="preserve">
+	<g>
+		<path class="path fill primary" d="M13,36H9V1c0-0.553-0.448-1-1-1S7,0.447,7,1v35H3c-0.552,0-1,0.447-1,1v12c0,0.553,0.448,1,1,1h4v3c0,0.553,0.448,1,1,1
```

Comment:
Shall we indent this file using 4 spaces?

Reply:
The file is indented with 4 spaces and I think the github renderer is showing the incorrect indent.